live-examples/html-examples/input/button.html

```diff
@@ -0,0 +1,8 @@
+<fieldset>
+    <legend>Button</legend>
+    <form>
+      <p> These buttons work with access key modifiers too. For Chrome, try `Ctrl + Alt + p`(Mac) / `Alt + p`(Others).</p>
```

Comment:
@nnupoor I would recommend we do without these for now. We are going to add JavaScript support to the editor in another round of iteration. You can keep the `accesskey` attribute but, remove this sentence and then the `onclick` JS parts as well.